clients/client-java/src/test/java/com/datastrato/gravitino/client/TestRelationalCatalog.java

```diff
@@ -569,6 +569,93 @@ public void testCreatePartitionedTable() throws JsonProcessingException {
         ex3.getMessage().contains("\"columns\" field is required and cannot be empty"));
   }
 
+  @Test
+  public void testCreateIndexTable() throws JsonProcessingException {
+    NameIdentifier tableId = NameIdentifier.of(metalakeName, catalogName, "schema1", "index1");
+    String tablePath = withSlash(RelationalCatalog.formatTableRequestPath(tableId.namespace()));
+
+    ColumnDTO[] columns =
```

Comment:
please remove DTO

Reply:
@mchades please help to check this.

Reply:
He is testing the HTTP request here, so using DTO is OK, but it will be better to use DTOConveter to do that

Reply:
ok